File: servers/game/src/main/java/fr/rob/game/setup/application/AppSetup.java
package fr.rob.game.setup.application;

import fr.rob.game.setup.domain.Setup;

import java.security.PublicKey;

public class AppSetup implements Setup {

    private PublicKey publicKey;

    @Override
    public PublicKey getJWTPublicKey() {
        return publicKey;
    }

    @Override
    public void setJWTPublicKey(PublicKey publicKey) {
        this.publicKey = publicKey;
    }
}

File: servers/game/src/main/kotlin/fr/rob/game/application/setup/AppSetup.kt
package fr.rob.game.application.setup

import fr.rob.game.domain.setup.Setup
import java.security.PublicKey

class AppSetup : Setup {

    private lateinit var publicKey: PublicKey

    override fun getJWTPublicKey(): PublicKey = publicKey

    override fun setJWTPublicKey(key: PublicKey) {
        publicKey = key
    }
}